src/pyuvdata/uvdata/mir_parser.py

```diff
@@ -2100,7 +2100,13 @@ def _fix_acdata(self):
             data_arr = self.ac_data._data[field]
             export_data_arr = self.bl_data._data[field]
             for key, idx_arr in ac_groups.items():
-                data_arr[idx_arr] = np.median(export_data_arr[bl_groups[key]])
+                try:
```

Comment:
Suggest checking with `if not idx_arr in data_arr` instead of catching the `KeyError`.

Reply:
Changed in 6bd6834c148e163f4856966f43a0663f5a079450